### DISCUSSION TITLE
Raspberry pi 4B 8GB (Ubuntu desktop) debug.log file errorErr.DOUBLE_SPEND every seconds

### DISCUSSION THREAD

**Discussion by d33gan** [Chia-Network/chia-blockchain#6213] [Support]
Hello Chia support Team,
i have make much efforts to farm chia on my Raspberry pi 4B 8GB (Ubuntu Desktop), and after i look in the debug.log file i was a little bit confused much Errors and warnings.

Please can you look in this file and tell me what is to do and if anything is okay .... and i can win chia with this setup?

i have attached the .log file here:
[debug.log](https://github.com/Chia-Network/chia-blockchain/files/6565334/debug.log)

thank you so much for helping me and your effort!

Greatings Deegan

**Comment by tiny0little** [comment]
you have a bit more ERRORS and WARNINGS than usual, but logs look fine. if your node is in sync nothing to worry about

**Reply by d33gan** [reply]
Yes all is synced fine and in GUI all farming correctly.... only the debug.log file makes me a little bit nervous.....
Are you a chia developer?  when i you say all fine than i am not worry :-)
but what i dont want is to lose my time with it because of that i want that all works fine... i think you underdstandthis :-)

okay than i make more plots with my desktop computer and move it to my raspberry pi :-)

**Reply by asiansaga** [reply]
Sorry for jumping in, but can you farm on rpi4 with no node running on internal network?

**Reply by d33gan** [reply]
So when the Full Node is syncing than i dont can farm....

and i see in last days the full Node jumps from syncing to sync and than he must sync again and i hear thats not good anybody here that can help me?

**Comment by d33gan** [comment]
Hello Chia Support.... my raspberry pi ... jumps by full node to syncing..... so its not synced all the time thats not good.. please help me i have the latest update...

**Comment by BrandtH22** [comment]
@d33gan , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
